krizten/Memoire-WebClient
# #165528673 Add Entry Page UI

### What does this PR do?
- Build out 'add-entry' page
#### Description of Task to be completed?
- Convert the UI/UX design provided to markup.
- Ensure the add-entry page is responsive to screen sizes and matches the design provided.
#### How should this be manually tested?
- Clone this repository and checkout to the branch: ```ft/165528673/add-entry-page```
- Ensure [yarn](https://yarnpkg.com/lang/en/) is installed, then install the project dependencies by simply running ```yarn``` in the terminal.
- Start the project using ```yarn start```.
- Using a web browser, navigate to ```http://localhost:3000/entries/new```
#### What are the relevant pivotal tracker stories?
#165528673
#### Screenshots (if appropriate)
![add-entry-desktop](https://user-images.githubusercontent.com/29978455/58054436-7ca16f00-7b52-11e9-9cdd-6731814dad75.png)
![add-entry-image-upload-desktop](https://user-images.githubusercontent.com/29978455/58054453-84611380-7b52-11e9-9318-80d99de001ca.png)
![add-entry-mobile](https://user-images.githubusercontent.com/29978455/58054463-8aef8b00-7b52-11e9-9b10-308648bf314d.png)
![add-entry-image-upload-mobile](https://user-images.githubusercontent.com/29978455/58054476-9478f300-7b52-11e9-9bf3-70afab2dfbae.png)
